src/Sarif.Multitool/TransformCommand.cs

```diff
@@ -25,10 +25,15 @@ public int Run(TransformOptions transformOptions)
         {
             try
             {
-                transformOptions.OutputFilePath = CommandUtilities.GetTransformedOutputFileName(transformOptions);
+                // Only set --output-file if --inline isn't specified. ValidateOptions will check
+                // to make sure that exactly one of those two options is set.
+                if (!transformOptions.Inline)
+                {
+                    transformOptions.OutputFilePath = CommandUtilities.GetTransformedOutputFileName(transformOptions);
+                }
 
                 bool valid = ValidateOptions(transformOptions);
-                if (!valid) { return 1; }
+                if (!valid) { return Failure; }
```

Comment:
>Failure [](start = 37, length = 7)

Our convention in this code base is to use all caps for values that are constants like this. IOW, it is helpful to know that Failure isn't some mutable property somewhere. AnalyzeCommandBase follows this convention. Please use all caps here or start the process of advocating for changing the existing standard (in which case we need to fix other occurrences). In general, we should prefer to keep this code base self-consistent. It's fine to revisit previous style decisions. FWIW, I like SUCCESS and FAILURE because they (to me) clearly indicate they are a readable placeholder for these constants.

Reply:
I actually like ALL_CAPS for constants like this, and I'm aware of the usage in `AnalyzeCommandBase`, but it violates the .NET style guidelines. I propose to change `AnalyzeCommandBase`.

I'm not going to hold the merge for this; Iet's discuss today (I'll leave myself a note to bring it up).

---
In reply to: [316743133](https://github.com/microsoft/sarif-sdk/pull/1653#discussion_r316743133) [](ancestors = 316743133)